java/src/jmri/configurexml/StoreXmlConfigAction.java

```diff
@@ -75,8 +75,8 @@ public static File getFileCustom(JFileChooser fileChooser) {
         // check for possible overwrite
         if (file.exists()) {
             int selectedValue = JOptionPane.showConfirmDialog(null,
-                    "File " + file.getName() + " already exists, overwrite it?",
-                    "Overwrite file?",
+                    rb.getString("FileOverwriteWarning1") + " " + file.getName() + " " + rb.getString("FileOverwriteWarning2"),
```

Comment:
You should be able use ```Bundle.getMessage("FileOverwriteWarning", file.getName())``` instead of building a String here. Copy jmri/profile/Bundle.java to jmri/configurexml/Bundle.java and change the name field to null to make this work.